plugins/catalog-backend-module-gitlab/api-report.md

```diff
@@ -52,21 +52,4 @@ export class GitLabDiscoveryProcessor implements CatalogProcessor {
     emit: CatalogProcessorEmit,
   ): Promise<boolean>;
 }
-
-// @public
-export class GitlabOrgDiscoveryEntityProvider implements EntityProvider {
```

Comment:
This is also a breaking change; you can export a deprecated const of this too

Reply:
this would require to have a dependency on the other newly created module, though.

I'm wondering whether we couldn't support both at the same backend module. However, it may make sense to split these up (org data vs general entities based on catalog files in repositories).